crates/bevy_ui/src/stack.rs

```diff
@@ -15,6 +15,24 @@ pub struct UiStack {
     pub uinodes: Vec<Entity>,
 }
 
+#[derive(Default)]
+pub struct StackingContextCache {
```

Comment:
This needs docs if it's public. I would probably add docs for internal users and doc(hidden) this.